src/lib/View/RelationView.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\View;
+
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\Core\MVC\Symfony\View\BaseView;
+
+/**
+ * {@inheritdoc}
+ */
+class RelationView extends BaseView
+{
+    /** @var \eZ\Publish\API\Repository\Values\Content\Content|null */
+    private $content;
+
+    /** @var \eZ\Publish\API\Repository\Values\Content\Location|null */
+    private $location;
+
+    /** @var \eZ\Publish\API\Repository\Values\ContentType\ContentType|null */
+    private $contentType;
+
+    /** @var string|null */
+    private $apiException;
```

Comment:
IMHO this should be an exception instance. We might want to use exception data in the error handling.